src/Controller/Command.php

```diff
@@ -187,6 +187,41 @@ public function run(): Response
                     $this->failWhenNotAuthenticated($user, $path);
                     return (new CreateLink($this->storage, $user, $this->request, $this->factory))
                         ->execute();
+                case preg_match('/^\/upload\/(\w{9,}).*$/', $path, $matches) === 1:
```

Comment:
This RegEx is already used by all other routes. I agree that using a more restrictive RegEx makes sense. I will check if there is a reason why this is used and if it can be changed without causing any problems I will do it everywhere.

Reply:
Don't seems to cause any problems. Changed it.